src/services/chat-classifier.ts

```diff
@@ -18,7 +18,8 @@ export const getFactsFromChatSteps = (steps = []) =>
 
 export const getClassesFromRuleResults = ({ events }) =>
   events
-    .map(event => event && event.params && event.params.id)
+    .map(event => event && event.params && event.params.classes)
+    .reduce((acc, classes) => [...acc, ...classes])
```

Comment:
is this essentially a flatMap?

also, no default argument present, not sure if it matters (could acc be undefined at start?)